Makefile

```diff
@@ -541,18 +541,21 @@ lint-spell:
 
 lint: lint-js lint-sass lint-cpp lint-html lint-spell
 
-ELECTRON_MOCHA_OPTIONS=--recursive --reporter spec
+MOCHA_OPTIONS=--recursive --reporter spec
```

Comment:
Should these be kept separate? Or is it guaranteed that all `mocha` options will always be the same as the `electron-mocha` options?

Reply:
Its guaranteed. `electron-mocha` aims to remain 100% CLI compatible with `mocha`.